src/DataProtection/Cryptography.KeyDerivation/src/PBKDF2/Pbkdf2Util.cs

```diff
@@ -27,16 +28,15 @@ private static IPbkdf2Provider GetPbkdf2Provider()
             }
             else
             {
-#if NETSTANDARD2_0 || NET461
                 return new ManagedPbkdf2Provider();
+            }
 #elif NETCOREAPP
-                // fastest implementation on .NET Core for Linux/macOS.
-                // Not supported on .NET Framework
-                return new NetCorePbkdf2Provider();
+            // fastest implementation on .NET Core for Linux/macOS.
```

Comment:
Maybe we can remove "for Linux/macOS" if we are also relying on this being the fastest on Windows as well?